core/error.go

```diff
@@ -38,17 +37,12 @@ type Error struct {
 // Error() allows *core.Error to present the `error` interface.
 func (e *Error) Error() string {
 	var ret string
+	ret = e.desc + "\n"
```

Comment:
Do we need to change this file as part of this PR?  Weren't we planning to just get rid of this `core.Error` type entirely? :D

Reply:
we are but not in this PR, here is really to get rid of the ENV['CONTIV_DEBUG'] thing only